app/assets/stylesheets/_state-file.scss

```diff
@@ -247,6 +247,14 @@
       }
     }
 
+    .tight-ish-checkboxes {
```

Comment:
This change was made to make the checkboxes taller to meet the minimum click target size. CfA might want to consider changing the tight-checkboxes class